dashboard/app/controllers/hint_view_requests_controller.rb

```diff
@@ -0,0 +1,14 @@
+class HintViewRequestsController < ApplicationController
+  def create
+    @hint_view_request = HintViewRequest.create(
+      script_id: params[:script_id],
+      level_id: params[:level_id],
+      user_id: current_user.id,
```

Comment:
I believe that will cause an error and/or redirect to the sign in page.

Does it also make sense to ensure that we don't even try to hit this endpoint if there is no logged in user?